torch/_inductor/codegen/wrapper.py

```diff
@@ -1072,6 +1079,11 @@ def codegen_model_constructor(self):
                         f"outputs_info_[{idx}].shape.emplace_back({size}, {size}, nullptr);"
                     )
 
+            if V.graph.aot_mode:
+                self.prefix.writeline(
+                    "AOTInductorModelContainerSetConstants(constants_info_);"
```

Comment:
It's probably bad to invoke ```AOTInductorModelContainerSetConstants``` from AOTModel's constructor, because ```AOTInductorModelContainerSetConstants``` acquires GPU memory and load constants into the device memory. If we have multiple ```AOTInductorModel``` instances, it means that we would end up with multiple copies of constants. In contrast, we should have a single copy of the constants, which is shared by each ```AOTInductorModel``` instance via pointers. So, a good place to load constants would be in AOTInductorModelContainer's constructor.